frontend/src/pages/ReservationCalendar.tsx

```diff
@@ -96,14 +98,34 @@ function ReservationCalendar() {
 
   const removeReservation = async (removeInfo: EventRemoveArg) => {
     const { event } = removeInfo;
-    const res = await deleteReservation(Number(event.id));
-    if (res === `Reservation ${selectedReservationRef.current?.id} deleted`) {
-      closeReservationModalFn();
-      selectedReservationRef.current = null;
-    } else {
+    removeInfo.revert();
```

Comment:
tää toimii nyt mut tän ei varmaan pitäis olla tässä? muilta osin näyttää hyvältä

Reply:
Se on siinä ihan tarkoituksella. Jos sen ottaa pois siitä, varauskalenterissa varaus häviää, kun popup ilmestyy. Se kyllä palaa, jos peruuttaa popupista. Käyttäjää voi hämätä se, että varaus poistuu jo varmistaessa poistoa. En keksinyt mitään virhettäkään, ja samaa ongelmaa ei jostain syystä ollut timeslottien puolella. Lisäämällä 
```
removeInfo.revert();
```
funktion removeReservation alkuun korjasi asian.

Reply:
aika jännä, en kyl keksi miks se toimii timesloteis muttei tässä :D